Enable CORS headers http referee replace and middleware

melvinsoft opened the pull request.

melvinsoft commented:
![](https://github.trello.services/images/mini-trello-icon.png) [(2) PSU: Final steps to support site launch](https://trello.com/c/OYKDkfgg/1554-2-psu-final-steps-to-support-site-launch)

tkeemon commented on lms/envs/aws_appsembler.py:
```diff
@@ -104,3 +104,14 @@
     # removing the database alias
     del DATABASES['appsembler_usage']
+
+if FEATURES.get('ENABLE_CORS_HEADERS'):
```
Same as my comment in the other PR: 

Let's set a default value for this get method in case it's not correctly set in the env:

```
FEATURES.get('ENABLE_CORS_HEADERS', False)
```

melvinsoft commented:
@tkeemon Same as the other PR:

Thanks for reviewing this, I've added the default for the settings, and also I've removed the middleware, I know is recommended, but not mandatory, and breaking the external requests again, and working without it, so I think we can remove it.
